airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -400,14 +405,17 @@ public void testIncrementalSync() throws Exception {
     final ConfiguredAirbyteCatalog configuredCatalog = CatalogHelpers.toDefaultConfiguredCatalog(catalog);
     configuredCatalog.getStreams().forEach(s -> {
       s.withSyncMode(SyncMode.INCREMENTAL);
-      s.withDestinationSyncMode(DestinationSyncMode.APPEND);
+      s.withDestinationSyncMode(DestinationSyncMode.OVERWRITE);
```

Comment:
The *first* one is changing to overwrite so that it clears out any data that may have been left there by previous tests.  The second one changes back to append to test append functionality correctly.  This is not a problem with the destination being unable to handle appends; it's a problem with the destination using a container that takes a long time to recycle, so we're not recycling it.  None of the other tests have a problem with this because they all use overwrite mode; this one needs to begin on overwrite mode as well in order to give consistent results.

Reply:
Tests should clean themselves up. The test case should not be having to worry about this. In production it is totally reasonable that on the first run there is going to be an APPEND so it is valuable for the test to emulate that behavior.